pyproject.toml

```diff
@@ -132,7 +132,7 @@ ignore = [
 ]
 format.preview = true
 lint.preview = true
-[tool.ruff.per-file-ignores]
+[tool.ruff.lint.per-file-ignores]
```

Comment:
Please use the previous format, don't create new section.

Reply:
I'm not clear if I'm supposed to be addressing this; the conversation was marked as "resolved" before I even saw it.

In case I am, I'm not clear on what you're talking about above. I didn't create a new section; I just renamed an existing section.